Source/WebKit/UIProcess/AuxiliaryProcessProxy.h

```diff
@@ -168,7 +168,17 @@ class AuxiliaryProcessProxy
         Running,
         Terminated,
     };
-    State state() const;
+    inline State state() const
```

Comment:
I don't know if we really want to inline this. If we do, it should definitely be after the class definition, to avoid polluting the class scope with large functions.

Reply:
Inlining this makes huge help to reduce the warnings in other classes. I moved the function body below the class definition.